contrib/win32/openssh/vstsbuild.ps1

```diff
@@ -0,0 +1,135 @@
+[cmdletbinding(DefaultParameterSetName='Build')]
+param(
+    [Parameter(Mandatory,ParameterSetName='Build')]
+    [String]$Name="X64",
+    # full paths to files to add to container to run the build
+    [Parameter(Mandatory,ParameterSetName='packageSigned')]
+    [string]$BuildPath,
+    [Parameter(ParameterSetName='packageSigned')]
+    [string]$SignedFilesPath
+)
+
+$gitBinFullPath = (Get-Command -Name git).Source
+if (-not $gitBinFullPath)
+{
+    throw "Git is required to proceed. Install from 'https://git-scm.com/download/win'"
+}
+
+function Get-RepoFork
+{
+    [CmdletBinding()]    
+    param([string]$AccountURL="https://github.com/powershell", [string]$RepoFork, [string]$repoLocalPath, [string]$BranchName)
```

Comment:
It' better to remove the default value as we always pass it. It adds unnecessary confusion in long run.

Reply:
Removed